index.html

```diff
@@ -1,17 +1,130 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Capstone Project - Concert Page</title>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="index.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Lato"
+      rel="stylesheet"
+    />
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
     <script src="index.js"></script>
-</head>
-<body>
-  <header>
-    
-  </header>
-    
-</body>
-</html>
\ No newline at end of file
+
+    <title>Capstone Project - Concert Page</title>
+  </head>
+  <body>
+    <header>
+      <div class="upper-header">
+        <ul class="list1">
+          <li><img src="./assets/fb icon grey.png" alt="facebook-link" class="header-top-img"/></li>
+          <li><img src="./assets/twitter icon grey.png" alt="twitter-link" class="header-top-img"/></li>
+          <li><p>English</p></li>
+          <li><p>My Page</p></li>
+          <li><p>Logout</p></li>
+        </ul>
+      </div>
+      <nav>
+        <a href="index.html"><img src="./assets/Logo.webp" alt="logo" id="logo"/></a>
+        <ul class="list2">
+          <li class="desktop-nav"><a href="about.html">About</a></li>
+          <li class="desktop-nav"><a href="#">Program</a></li>
+          <li class="desktop-nav"><a href="#">Join</a></li>
+          <li class="desktop-nav"><a href="#">Sponsor</a></li>
+          <li class="desktop-nav"><a href="#">News</a></li>
+          <li class="hamburger">
+            <img src="./assets/hamburger.png" alt="hamburger" id="hamburger"/>
+          </li>
```

Comment:
- [ ] The mobile navbar looks great, however in the guidelines the hamburger button is on the left and there is no logo, kindly update these minor details to follow the design guidelines 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/204088163-40896c40-98ef-4e7c-ba00-041e58146088.png)

 ## Guidelines:
![image](https://user-images.githubusercontent.com/50721479/204088180-14bdd268-c83d-4001-af2f-d7aa0ce8066c.png)